app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -122,13 +124,15 @@
       <li>a parent or carer</li>
       <li>disabled</li>
     </ul>
-    <p>
-      Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.
-    </p>
     <p>
       Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.
     </p>
-
+  <%= render Content::ExpanderComponent.new(
+    title: "bursaries and scholarships",
```

Comment:
@emmaachesongray I wonder if this title needs to be broader given that the information above is wider than just bursaries and scholarships - and the information in the expander/on the destination page is similarly broader?

(reword as needed!!)

```suggestion
    title: "funding your training",
```